### PR TITLE
Two help pages, updated header nav menu, id page notes/tips

### PR DESCRIPTION
**Pull request by ahiatt14**
This isn't every help page, but since nothing here touches the database, or has any requests & data interaction, figured I'd keep this a separate PR. This incorporates all the help page content from that google doc except the "non-gall faq"/non-gall structure guide, for issue #60

### CODE REVIEW

File: pages/filterguide.tsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        <Accordion>
+          <Card>
+            <Card.Header>
+              <Accordion.Toggle as={Button} variant="light" eventKey="0">
+                Location
+              </Accordion.Toggle>
+            </Card.Header>
+            <Accordion.Collapse eventKey="0">
+              <ListGroup>
+                <Item key="at leaf vein angles">
```
Comment (jeffdc) [inline pages/filterguide.tsx:26]: We should sort these alpha I think.

Comment (jeffdc) [reply pages/filterguide.tsx:26]: Or group them by general plant area? Leaves, stems, roots? @Megachile

**Review by jeffdc — COMMENTED** [review]
LGTM. I have a bunch of changes to the id page so I will have to manually merge those changes into mine, but that is fine.

File: pages/filterguide.tsx (hunk 1 above)
Comment (Megachile) [reply pages/filterguide.tsx:26]: Either way seems fine to me.

**Pull request merged**